spec/spec_helper.rb

```diff
@@ -91,3 +91,15 @@ def self.included(context)
   config.formatter = 'documentation'
   config.color = true
 end
+
+if ENV['TEST_OTEL'] == 'true'
```

Comment:
also check if Ruby version >= 3.0? OTel requires version >= 3

Reply:
Since 2.7 is already end of life'd, and we support only maintained Ruby versions, it wouldn't be necessary. I will update [the value on the gemspec file](https://github.com/elastic/elastic-transport-ruby/blob/main/elastic-transport.gemspec#L45) though to specify it requires `>= 3.0`.

Reply:
ok, that simplifies things. Thanks!

Reply:
just documenting here that we can't drop 2.7 for now, so we'll keep in the checks for ruby version